common/src/main/resources/common/destroy-data.feature

```diff
@@ -4,7 +4,7 @@ Feature: destroy data for tenant
     * url baseUrl
     * configure readTimeout = 90000
     * configure retry = { count: 5, interval: 5000 }
-    * callonce login admin
```

Comment:
why does this need to change? This is in the common module, so we have to be careful here.

Reply:
because it was throwing 401 error at clean-up process